vcbuild.bat

```diff
@@ -36,6 +36,10 @@ set release_urls_arg=
 set build_release=
 set enable_vtune_arg=
 set configure_flags=
+set build_addons=
+
+:: assign path to node_exe
+set node_exe="%config%\node.exe"
```

Comment:
The quotes should not be stored in the variable, but added to every place where the variable is used, to be consistent with all other variables in this file.